packages/gafl-webapp-service/src/pages/renewals/renewal-inactive/__tests__/route.spec.js

```diff
@@ -1,45 +1,225 @@
+        cache: () => ({
+          helpers: {
+            status: {
+              getCurrentPermission: mockStatusCacheGet
+            }
+          }
+        }),
+        i18n: {
+          getCatalog: jest.fn(() => englishTranslations)
+        },
+        locale: ''
+      })
+
+      const { bodyMessage } = getTitleAndBodyMessage(englishTranslations, reason, referenceNumber, formatSymbol)
+
+      expect(titleAndBodyMessage.bodyMessage).toEqual(bodyMessage)
+    })
+
+    it.each([['2020-12-13T23:59:59Z'], ['2020-01-13T23:59:59Z'], ['2020-10-13T23:59:59Z']])(
+      'endDate is passed to moment',
+      async endDate => {
+        mockStatusCacheGet.mockImplementationOnce(() => ({
+          authentication: { reason: RENEWAL_ERROR_REASON.NOT_DUE, endDate },
+          referenceNumber: 'ABC123'
+        }))
+        moment.mockImplementation(getMomentMockImpl())
+        await getData(mockRequest)
+        expect(moment).toHaveBeenCalledWith(endDate, 'YYYY-MM-DD', 'en-gb')
+      }
+    )
+
+    it('cacheDateFormat being what is expected', async () => {
       mockStatusCacheGet.mockImplementationOnce(() => ({
         authentication: { reason: RENEWAL_ERROR_REASON.NOT_DUE, endDate: '2020-12-13T23:59:59Z' },
         referenceNumber: 'ABC123'
       }))
-      const result = await getData(mockRequest)
-      expect(result.bodyMessage).toBe('The licence ending in ABC123 does not expire until 13 December 2020')
+      moment.mockImplementation(getMomentMockImpl())
+      await getData(mockRequest)
+      expect(moment).toHaveBeenCalledWith('2020-12-13T23:59:59Z', 'YYYY-MM-DD', 'en-gb')
+    })
+
+    it.each([['en-gb'], ['cy']])('locale is set on moment, to whatever the request.locale is', async language => {
+      const mockRequest = {
+        cache: () => ({
+          helpers: {
+            status: {
+              getCurrentPermission: mockStatusCacheGet
+            }
+          }
+        }),
+        i18n: {
+          getCatalog: jest.fn(() => englishTranslations)
+        },
+        locale: language
+      }
+      mockStatusCacheGet.mockImplementationOnce(() => ({
+        authentication: { reason: RENEWAL_ERROR_REASON.NOT_DUE, endDate: '2020-12-13T23:59:59Z' },
+        referenceNumber: 'ABC123'
+      }))
+      const format = jest.fn()
+      moment.mockImplementation(getMomentMockImpl({ format }))
+      await getData(mockRequest)
+      expect(moment).toHaveBeenCalledWith('2020-12-13T23:59:59Z', 'YYYY-MM-DD', language)
+    })
+
+    it('dateDisplayFormat is passed to format', async () => {
+      mockStatusCacheGet.mockImplementationOnce(() => ({
+        authentication: { reason: RENEWAL_ERROR_REASON.NOT_DUE, endDate: '2020-12-13T23:59:59Z' },
+        referenceNumber: 'ABC123'
+      }))
+      const format = jest.fn()
+      moment.mockImplementation(getMomentMockImpl({ format }))
+      await getData(mockRequest)
+      expect(format).toHaveBeenCalledWith('D MMMM YYYY')
```

Comment:
I think the same comments apply here - focus in the `expect` statements on a single argument, using `expect.any(type)`.